components/values/ValueContact.vue

```diff
@@ -0,0 +1,76 @@
+<script setup lang="ts">
+const { t } = useI18n();
+
+const contacts = [
+  {
+    title: 'Email',
+    description: t('values.contact_section.email.description'),
+    href: 'mailto:info@rotki.com',
+    icon: 'chat-smile-2-line',
```

Comment:
Ok so instead of copy pasting the urls/mail addresses do you think we can move them instead to config?